src/epgsql_wire.erl

```diff
@@ -209,6 +209,6 @@ encode_list(L) ->
     <<(byte_size(Bin)):?int32, Bin/binary>>.
 
 encode_standby_status_update(ReceivedLSN, FlushedLSN, AppliedLSN) ->
-    {MegaSecs, Secs, MicroSecs} = erlang:now(),
```

Comment:
I figured as much.  I recall reading about some of the time changes - are you positive that that's the best call and not one of the monotonic ones?  It certainly works with all versions.